uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1652,6 +1653,81 @@ sub computeSNR {
     }
 }
 
+################################################################
+#######################  computeDeepQC #############################
+################################################################
+sub computeDeepQC {
+    my $this = shift;
+    my ($row, $filename, $fileID, $base, $fullpath, $message);
+    my ($tarchiveID, $tarchive_srcloc, $profile)= @_;
+    my $data_dir = NeuroDB::DBI::getConfigSetting(
+                        $this->{dbhr},'dataDirBasepath'
+                        );
+    my $upload_id = getUploadIDUsingTarchiveSrcLoc($tarchive_srcloc);
+    my $ua = LWP::UserAgent->new;
+    my $json = 'JSON::PP'->new;
+    # note: url must be changed to production version later
+    # note 2: include DeepQC modality in prod file
+    my $acqIDForT1Scan = 44;
+    my $query = "SELECT FileID, file, AcquisitionProtocolID from files f ";
+    my $where = "WHERE f.TarchiveSource=?";
+    $query = $query . $where;
+
+    if ($this->{debug}) {
+        print $query . "\n";
+    }
+    my $minc_file_arr = ${$this->{'dbhr'}}->prepare($query);
+    $minc_file_arr->execute($tarchiveID);
+
+    while ($row = $minc_file_arr->fetchrow_hashref()) {
+        $filename = $row->{'file'};
+        my $fileID = $row->{'FileID'};
+        my $acqID = $row->{'AcquisitionProtocolID'};
+        $base = basename($filename);
+        $fullpath = $data_dir . "/" . $filename;
+        if (-e $fullpath) {
+            if ($acqID == $acqIDForT1Scan) {
+                my $url = 'http://127.0.0.1:5000/deepqc/' . $fileID;
```

Comment:
is this url specific for IBIS? If so, might want to find a way to make this more generic too. Is the code from Andrew generic enough to work on any T1W?

Reply:
@cmadjar To my understanding, he's only had success validating his model on IBIS data.

Reply:
"127.0.0.1" is specific to the server that it's running on, not specific to ibis.